client/src/components/CartDrawer.tsx

```diff
@@ -0,0 +1,54 @@
+import { FaPlus, FaMinus } from 'react-icons/fa';
+import { toDollars } from '../library/to-dollars';
+import { CartContext } from './CartContext';
+import { useContext } from 'react';
+
+type Props = {
+  isItOpen: boolean;
+  close: () => void;
+};
+
+export function CartDrawer({ isItOpen, close }: Props) {
+  const { cart } = useContext(CartContext);
+
+  return (
+    <>
+      {isItOpen && (
```

Comment:
I'd move this one outside the `return` statement, as mentioned in the previous PR.